lib/WorseReflection/Core/SourceCodeLocator/NativeReflectionFunctionSourceLocator.php

```diff
@@ -22,17 +23,22 @@ public function locate(Name $name): SourceCode
         ));
     }
 
-    private function sourceFromFunctionName(Name $name)
+    private function sourceFromFunctionName(Name $name): SourceCode
     {
         $function = new ReflectionFunction($name->__toString());
 
+        $fileName = $function->getFileName();
         if ($function->isInternal()) {
             throw new SourceNotFound(sprintf(
                 'Function "%s" is an internal function, there is another locator for that',
                 $name->__toString()
             ));
         }
 
-        return SourceCode::fromPathAndString($function->getFileName(), file_get_contents($function->getFileName()));
```

Comment:
can use `SourceCode::fromPath($function>getFilename())` instead

Reply:
Yep, I have checked the whole code base and used it wherever possible.